### PR TITLE
HAWQ-1191. Decomissioned DELIMITER from HiveOrc profile.

### PR DESCRIPTION
**Pull request by sansanichfb**

### CODE REVIEW

**Comment by shivzone** [comment]
I'm a bit concerned about the impact of removing this function parseDelimiterChar completely without completely understand the impact of it w.r.t Partition filtering. Lets merge this change once we verify that Partition based filtering works fine with all the Hive related profiles

File: pxf/pxf-hive/src/main/java/org/apache/hawq/pxf/plugins/hive/HiveResolver.java [hunk 1]
```diff
@@ -79,7 +79,7 @@
     private String serdeName;
     private String propsString;
     String partitionKeys;
-    char delimiter;
+    char delimiter = 1;
```
Comment (kavinderd) [inline pxf/pxf-hive/src/main/java/org/apache/hawq/pxf/plugins/hive/HiveResolver.java]: Why do we even need this value anymore?

**Comment by kavinderd** [comment]
So the delimiter we were previously explicitly asking for in the DDL will just be retrieved internally?

**Comment by sansanichfb** [comment]
@shivzone, @kavinderd I double checked for HiveRc and HiveText - they still need delimiter, which should be exposed to users, so in this PR I just removed DELIMITER for HiveOrc profile.

**Pull request closed without merging**